.openshift-ci/get_latest_helm_chart_versions.py

```diff
@@ -165,12 +192,22 @@ def remove_helm_repo():
 
 
 def run_command(command, log_stdout=True):
-    result = subprocess.run(command, shell=True, encoding='utf-8',
-                            stdin=subprocess.DEVNULL, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-
-    stdout = format_command_output("Stdout", result.stdout) if log_stdout else ""
+    result = subprocess.run(
+        command,
+        shell=True,
+        encoding="utf-8",
+        stdin=subprocess.DEVNULL,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=False
```

Comment:
```suggestion
```
This is the default, no?

Reply:
It is. I think the motivation is that new python users would not expect that to be the case: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/subprocess-run-check.html